### PR TITLE
BUGZ-516 - add OS type to connect message for logging

### PR DESCRIPTION
**Pull request by roxanneskelly**
https://highfidelity.atlassian.net/browse/BUGZ-516

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20045/HighFidelity-Beta-PR15799-2ed5e8a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20045/HighFidelity-Beta-Interface-PR15799-2ed5e8a.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20045/HighFidelity-Beta-PR15799-2ed5e8a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20045/HighFidelity-Quest-Beta-PR15799-2ed5e8a.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20045/HighFidelity-Beta-PR15799-2ed5e8a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20045/HighFidelity-Beta-Interface-PR15799-2ed5e8a.exe

File: domain-server/src/DomainGatekeeper.cpp [hunk 1]
```diff
@@ -127,7 +127,8 @@ void DomainGatekeeper::processConnectRequestPacket(QSharedPointer<ReceivedMessag
             << "and machine fingerprint" << nodeConnection.machineFingerprint 
             << "user" << username 
             << "reason" << QString(nodeConnection.connectReason ? "SilentDomainDisconnect" : "Connect")
-            << "previous connection uptime" << nodeConnection.previousConnectionUpTime/USECS_PER_MSEC << "msec";
+            << "previous connection uptime" << nodeConnection.previousConnectionUpTime/USECS_PER_MSEC << "msec"
+            << "\nsysinfo" << nodeConnection.SystemInfo;
```
Comment (howard-stearns) [inline domain-server/src/DomainGatekeeper.cpp]: why the \n?  I'm not sure what effect that has on splunk.

Comment (roxanneskelly) [reply domain-server/src/DomainGatekeeper.cpp]: helped with readability in the logs.  We can remove it or look at see what happens in splunk.

Comment (roxanneskelly) [reply domain-server/src/DomainGatekeeper.cpp]: meh, went ahead and removed it anyway

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20047/HighFidelity-Beta-PR15799-0576e6a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20047/HighFidelity-Beta-Interface-PR15799-0576e6a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20047/HighFidelity-Beta-PR15799-0576e6a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20047/HighFidelity-Beta-Interface-PR15799-0576e6a.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20047/HighFidelity-Beta-PR15799-0576e6a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20047/HighFidelity-Quest-Beta-PR15799-0576e6a.apk)

**Pull request merged**